port/unix/omrintrospect.c

```diff
@@ -1685,7 +1681,7 @@ omrintrospect_threads_nextDo(J9ThreadWalkState *state)
 		goto cleanup;
 	}
 
-#if defined(LINUX) || defined(AIXPPC) || defined(J9ZOS390)
```

Comment:
To clarify, Linux now uses sem_timedwait_r() rather than the busy loop with sem_trywait_r() and sigqueue().